controllers/submariner/submariner_controller.go

```diff
@@ -55,8 +55,10 @@ import (
 )
 
 const (
-	gatewayMetricsServerPort   = 8080
-	globalnetMetricsServerPort = 8081
+	gatewayMetricsServicePort   = 8080
+	globalnetMetricsServicePort = 8081
+	gwMetricsServerPort         = "32780"
+	gnMetricsServerPort         = "32781"
```

Comment:
```suggestion
	gatewayMetricsServerPort         = "32780"
	globalnetMetricsServerPort         = "32781"
```

For consistency, also `gw/gn` look very similar when reading the code which could lead to confusion.

Reply:
It was already merged: https://github.com/submariner-io/submariner/pull/1967 I didn't add Depend On coz it was merged by the time I raised this one. Agree with your suggestion, makes sense as these are for two separate pods altogether. Want me to raise another PR on submariner to change it?